### PR TITLE
Progress bar

### PR DESCRIPTION
**Pull request by kwood15**
Please review as per story #46

Screenshots here:

screen shot 2017-09-19 at 17 27 36
screen shot 2017-09-19 at 17 27 55

### CODE REVIEW

**Comment by rbrtsmith** [comment]
A single progress bar design should be sufficient.  (Alpha colour).  I don't see any needs for variations on this.  Again it adds weight to the CSS file and it's more stuff to maintain.

I'm not too keen on the 3d effect, the rest of the design in Nebula is flat so we should keep that consistency - can be achieved by dead stops between the colours.

We also won't need a label for the progress bar.  Consumers can add labels to the outside themselves if necessary.  Also would be good to see how it looks with a less intense colour variation i.e. rather than colour alpha and white try colour -alpha and `lighten($color-alpha, 15%)` for example.

**Comment by kwood15** [comment]
Have updated as per comments above, screenshots here:

screen shot 2017-09-21 at 14 57 53

screen shot 2017-09-21 at 14 52 22

File: nebula-components/components/_progress-bar.scss [hunk 1]
```diff
@@ -0,0 +1,12 @@
+@if $nb-use-progress-bar {
+  $nb-color-alpha-light: lighten($nb-color-alpha, 15%);
+
+  .#{$nb-namespace}c-progress-bar {
+    border: 1px solid $nb-color-base-border;
+  }
+
+  .#{$nb-namespace}c-progress-bar__indicator {
+    height: ($nb-spacing-unit-double - $nb-spacing-unit-half);
+    background: repeating-linear-gradient(-45deg, $nb-color-alpha-light, $nb-color-alpha-light $nb-spacing-unit, $nb-color-alpha $nb-spacing-unit, $nb-color-alpha $nb-spacing-unit-double); //linear-gradient(to bottom, $nb-color-grey-v-light, $nb-color-info);
```
Comment (rbrtsmith) [inline nebula-components/components/_progress-bar.scss]: Just need to remove the comment at the end of the line.

Comment (kwood15) [reply nebula-components/components/_progress-bar.scss]: Done

**Review by rbrtsmith — APPROVED** [review]

**Pull request merged**